tidb-binlog/binlogctl/README.md

```diff
@@ -54,10 +54,14 @@ Then the result will be like this:
 we would format output later :)
 
 
-### unregister pump/drainer
+### update pump/drainer's state
+pump/drainer's state can be online, pausing, paused, closing and offline. In most cases, we only need update pump/drainer's state to paused or offline.
 ```
-bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd delete-pump/delete-drainer -node-id ip-127-0-0-1:8250/{nodeID}
+bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd update-pump/update-drainer -node-id ip-127-0-0-1:8250/{nodeID} -state {state}
```

Comment:
can we provoid more simple ways, like offline drainer, pause drainer rather than update state

Reply:
done, add cmd pause-pump, pause-drainer, offline-pump and offline-drainer